data/processors/structure.py

```diff
@@ -104,6 +104,7 @@ def _get_type(_id, _data):
             "building": _("Gebäude"),
             "room": _("Raum"),
             "virtual_room": _("Raum/Gebäudeteil"),
+            "poi": _("POI"),
```

Comment:
I have not tested this, but does this work (given that `en`=`de` in this case):
```suggestion
            "poi": "POI",
```